src/Psalm/Internal/PhpVisitor/TraitFinder.php

```diff
@@ -71,6 +78,10 @@ public function getNode(): ?PhpParser\Node\Stmt\Trait_
 
         try {
             $reflection_trait = new ReflectionClass($this->fq_trait_name);
+
+            if ($this->config->isReflectionFromSupportedExtension($reflection_trait)) {
+                return null;
+            }
```

Comment:
I'm not too sure about this one, but I don't know of any extensions that define traits. I suppose if it's not working correctly it can be fixed if/when such an extension is added.